.eslintrc

```diff
@@ -5,13 +5,10 @@ ecmaFeatures:
   - generators
 env:
   node: true
+  es6: true
```

Comment:
How many of the settings changes are necessary to accommodate the changes to the build? Would it be possible to pull them into the other PR and drop them from this one?